subsys/partition_manager/Kconfig

```diff
@@ -111,6 +111,10 @@ config PM_SINGLE_IMAGE
 
 DT_CHOSEN_EXT_FLASH:= nordic,pm-ext-flash
 
+config PM_EXTERNAL_FLASH_HAS_DRIVER
+	bool
+	default y if NORDIC_QSPI_NOR
```

Comment:
```suggestion
	default y if NORDIC_QSPI_NOR || SPI_NOR
```

Reply:
Not a fan of this, this is going to need an ever growing list of every type of external flash. Currently people are using QSPI flash, NOR flash, EEPROM, NAND flash, filesystem on external device, etc.

Reply:
Do we have a better idea? Supporting at least a subset of devices without user's intervention is still better than supporting QSPI_NOR only, but if there's a better solution we definitely should go for it :)